src/components/components/PercentageLine.js

```diff
@@ -0,0 +1,143 @@
+import React, { useRef } from 'react'
+import handleViewport from 'react-in-viewport'
+import styled from 'styled-components'
+import { formatCount, formatPercentage } from '../Util'
+
+// ----------------------------------------------------------------------------
+// Const
+// ----------------------------------------------------------------------------
+
+const RISKS = [
+  {
+    displayName: `Висок риск`,
+    name: `High`,
+    color: `#9C1414`,
+  },
+  {
+    displayName: `Среден риск`,
+    name: `Medium`,
+    color: `#F7B00D`,
+  },
+  {
+    displayName: `Нисък риск`,
+    name: `Low`,
+    color: `#0a7399`,
+  },
+]
+
+// ----------------------------------------------------------------------------
+// Utils
+// ----------------------------------------------------------------------------
+
+const generateTooltip = (color, riskName, percentage, count) => {
```

Comment:
Utils like these could be extracted into separate small files when appropriate to reduce the clutter a bit.

Reply:
Agreed. Will refactor separately, if we want to release this now though.